rllib/env/vector_env.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 
-@PublicAPI
+@OldAPIStack
```

Comment:
Imo we should keep this until we have a better solution to vectorize multi-agent environments as `pettingzoo` doe snot provide vectorization, yet.